nfc/NfcCx/Cx/NfcCxSE.cpp

```diff
@@ -20,6 +20,9 @@ Module Name:
 
 #include "NfcCxSE.tmh"
 
+#define NFCCX_SE_FLAG_SET_POWER_REFERENCE 0x1
+#define NFCCX_SE_FLAG_SET_ROUTING_TABLE 0x2
```

Comment:
These macros are neither from Spec or Proprietary. its the existing macro in the LIBNFC which has been reused